src/invoices/InvoiceLineForm/InvoiceLineForm.js

```diff
@@ -124,6 +127,7 @@ class InvoiceLineForm extends Component {
     const totalAmount = calculateTotalAmount(formValues);
     const isEditPostApproval = IS_EDIT_POST_APPROVAL(initialValues.id, initialValues.invoiceLineStatus);
     const isDisabledToEditAccountNumber = Boolean(isEditPostApproval || (poLineId && accountNumber));
+    const isDisabledEditFundDistribution = [INVOICE_STATUS.approved, INVOICE_STATUS.paid].includes(invoice.status);
```

Comment:
is it fair to rely on invoice status, and not on invoice line?

Reply:
please use `IS_EDIT_POST_APPROVAL` with invoice

Reply:
Updated